docs/components/board/pi.md

```diff
@@ -1,25 +1,27 @@
 ---
-title: "Configure a pi board"
+title: "Configure a Raspberry Pi 4, 3, or Zero 2 board"
```

Comment:
:+1: Thanks for specifying that these are for *raspberry* pis! Olivia and I are working on orange pi, rock pi, and eventually banana pi, and none of them will use this component.